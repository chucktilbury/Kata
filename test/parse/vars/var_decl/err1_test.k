;------------------------------------------------------------------------------
; Expected a symbol
;------------------------------------------------------------------------------

var const float some_name ; pass
var float 0.0 ; fail