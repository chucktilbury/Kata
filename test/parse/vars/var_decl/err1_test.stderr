Syntax: /home/chuck/Src/Kata/test/parse/vars/var_decl/err1_test.k: 6: 14: expected a symbol but got literal float
Syntax: /home/chuck/Src/Kata/test/parse/vars/var_decl/err1_test.k: 6: 14: expected a variable declaration but got literal float
Syntax: /home/chuck/Src/Kata/test/parse/vars/var_decl/err1_test.k: 6: 14: expected module item or end of file but got var
Syntax: /home/chuck/Src/Kata/test/parse/vars/var_decl/err1_test.k: 6: 14: module cannot be empty
